prod/templates/config/beacon-relay.json

```
@@ -0,0 +1,25 @@
+{
+  "source": {
+    "beacon": {
+      "endpoint": "http://127.0.0.1:9596",
+      "stateEndpoint": "http://127.0.0.1:9596",
```

```suggestion
      "endpoint": "https://ec2-18-197-55-65.eu-central-1.compute.amazonaws.com:9596",
      "stateEndpoint": "https://ec2-18-197-55-65.eu-central-1.compute.amazonaws.com:9596",
```

Need to setup security group to allow the relayer instance to access lodestar service.

---

Just an FYI, both the client and server are in the same DC, so from a networking perspective its cheaper and more performant to use the internal DNS name ip-172-31-42-147.eu-central-1.compute.internal which is only resolvable inside the datacentre.